arrow/compute/exprs/exec_test.go

```diff
@@ -135,8 +137,12 @@ func TestComparisons(t *testing.T) {
 		one  = scalar.MakeScalar(int32(1))
 		two  = scalar.MakeScalar(int32(2))
 
-		str = scalar.MakeScalar("hello")
-		bin = scalar.MakeScalar([]byte("hello"))
+		str           = scalar.MakeScalar("hello")
+		bin           = scalar.MakeScalar([]byte("hello"))
+		exampleUUID   = uuid.MustParse("102cb62f-e6f8-4eb0-9973-d9b012ff0967")
+		uidStorage, _ = scalar.MakeScalarParam(exampleUUID[:],
+			&arrow.FixedSizeBinaryType{ByteWidth: 16})
+		uid = scalar.NewExtensionScalar(uidStorage, extensions.NewUUIDType())
```

Comment:
`uid` -> `uuid`?

Reply:
`uuid` is used as the name of the `uuid` package, so I'd end up with a conflict / issue if I use the name for the variable here. I can rename it as something else though if we prefer.

Reply:
Ah! `uuidStorage` and `uuidScalar` may be better. (I feel that `uid` is "user id"...)

Reply:
will do!